src/Framework/UnitTests/CustomEventArgSerialization_Tests.cs

```diff
@@ -558,6 +570,7 @@ public void TestProjectFinishedEventArgs()
             Assert.IsTrue(string.Compare(genericEvent.ProjectFile, newGenericEvent.ProjectFile, StringComparison.OrdinalIgnoreCase) == 0, "Expected ProjectFile to Match");
         }
 
+#if FEATURE_BINARY_SERIALIZATION
```

Comment:
It looks like this #if and the following one are no longer necessary since the entire file is disabled if FEATURE_BINARY_SERIALIZATION isn't defined.